app/src/cards/CardWrapper.tsx

```diff
@@ -47,15 +49,32 @@ function CardWrapper(props: {
             {optionalTitle}
             {props.children()}
             {!props.hideFooter && (
-              <CardContent>
-                <LinkWithStickyParams
-                  target="_blank"
-                  to={`${DATA_CATALOG_PAGE_LINK}?${DATASET_PRE_FILTERS}=${props.datasetIds.join(
-                    ","
-                  )}`}
-                >
-                  View Data Sources
-                </LinkWithStickyParams>
+              <CardContent className={styles.CardFooter}>
+                Sources:{" "}
```

Comment:
This feels a bit verbose to me. What do you think of doing this?

Data from Covid Tracking Project (updated Dec 10) and American Community Survey (updated Dec 20)

Obviously need to check with Chi, and fine to submit as is and iterate later

Reply:
adding to frontend sync!